style.css

```diff
@@ -132,8 +141,511 @@ header {
 }
 
 .image .img1 {
+  margin-left: 32px;
   height: 300px;
   top: -30px;
   bottom: 0.36%;
   overflow: hidden;
 }
+
+/* == work section start == */
+
+.grid {
+  display: grid;
+  width: 375px;
```

Comment:
- [  ] :thinking: I think it would be better to use percentages and vw to make your site fit the screens because all mobile phones do not have the same width. :recycle: